system_baseline/openapi/internal_api.spec.yaml

```diff
@@ -102,6 +147,17 @@ components:
       items:
         type: string
         format: uuid
+    SystemIdsList:
```

Comment:
why do we have this object instead of just the array below?

Reply:
for the sake of consistency as we have it already this way in this repo